MultiplexReservationSystem/demo.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+dates="2023-04-19T11:00,2023-04-20T11:00"
+pageNum=0
+
+echo
+echo
+echo
+
+echo -e "\n\033[0;32mSystem lists available screenings: \033[0m"
+
+data=$(curl -s http://localhost:8080/screenings?page=0) &&
```

Comment:
Co dają nam te `&&` na końcu?

Reply:
źródło: https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_10
operator '&&' oznacza AND_IF, czyli jeśli polecenie się powiedzie, to wykonaj drugie polecenie. Na początku miałem inny plan z przeprowadzaniem use case'a. Po refaktoryzacji ten operator pozostał na końcu linii pierwszego cURLa, a reszta cURLów, nazwjimy to ładnie, dziedziczyła z właśnie tego pierwszego ;>

Reply:
:D Ubawiłeś mnie :)